src/controlles/productController.js

```diff
@@ -0,0 +1,92 @@
+const { join } = require("path");
+const productService = require("../services/productService");
+const { json } = require("express");
+// 장바구니 상품 목록 호출
+const shoppingItems = async (req, res) => {
+    try {
+        const userInfo = req.user
+        const result = await productService.shoppingCart(userInfo);
+        return res.status(202).json({message: result});
+    } catch(error) {
+        return res.status(400).json({ message: "SHOPPINGITEMS ERROR", error });
+    }
+};
+// 상품 개수 적용 시키는 
+const itemUpdate = async (req, res) => {
```

Comment:
```suggestion
const updateCartItemQuantity = async (req, res) => {
```
함수 이름에서부터 어떤 동작을 수행하는지 명확히 드러나도록 명명하시는 것이 좋습니다